src/pages/404.tsx

```diff
@@ -0,0 +1,152 @@
+import { Box, Flex, Image, Text, useMediaQuery } from '@chakra-ui/react';
+
+import Footer from '@/uikit/Footer';
+import Header from '@/uikit/Header';
+
+const descriptions = {
+  mobile: () => (
+    <Text as='h3' fontWeight='light'>
+      Whoops, you’ve wandered far, even Pluto is in your rearview! Seems like
+      the page you’re looking for has drifted off-course
+    </Text>
+  ),
+  desktop: () => (
+    <>
+      <Text as='h3' fontWeight='light'>
+        Whoops, you’ve wandered far, even Pluto is in your rearview!
+      </Text>
+      <Text>Seems like the page you’re looking for has drifted off-course</Text>
+    </>
+  ),
+};
+
+export default function Page404() {
+  const [isMobile] = useMediaQuery('(max-width: 768px)');
+  const [isDesktop] = useMediaQuery('(min-width: 1024px)');
```

Comment:
there is missing screen for tablet width (768 - 1023). actually I prefer just using 1 breakpoint (mobile), and the the rest will use above mobile breakpoint

so the logic in the component render will be like this

```
isMobile? <>component for mobile</> : <>component for tablet and above</>
```

Reply:
or event better, you can use `useBreakpointValue()` so we doesnt need to set the max-width manually and let it use from the chakra-ui standard

https://v2.chakra-ui.com/docs/hooks/use-breakpoint-value

ex: 
``` const isMobile = useBreakpointValue({base: true, md: false})```

Reply:
i don't think about `isMobile? <>component for mobile</> : <>component for tablet and above</>` 

since the goal is to reduce the way how we read the code XD

```ts
{isDesktop && (
          <>
            {/* planet left */}
            <Image
              src='/assets/images/planet_1.svg'
              alt='planet'
              position='absolute'
              left={{ base: 'none', md: '-105' }}
              bottom={{ base: 'none', md: '-125' }}
              overflow='hidden'
              zIndex={3}
              loading='lazy'
            />
            {/* planet right */}
            <Image
              src='/assets/images/planet_2.svg'
              alt='planet'
              w={{ base: '100%', md: '70%' }}
              position='absolute'
              right={{ base: 'none', md: '-125' }}
              top={{ base: 'none', md: '-105' }}
              overflow='hidden'
              zIndex={3}
              loading='lazy'
            />
            {/* Flare top */}
            <Image
              src='/assets/images/flare_2.svg'
              alt='flare'
              h='full'
              w='full'
              position='absolute'
              top={{ base: 'none', md: '-305', lg: '-105' }}
              zIndex={3}
              loading='lazy'
            />
            {/* Flare bottom */}
            <Image
              src='/assets/images/flare_1.svg'
              alt='flare'
              w='full'
              position='absolute'
              bottom={{ base: 'none', md: '-305' }}
              zIndex={3}
              loading='lazy'
            />
          </>
        )}

        {isMobile && (
          <>
            {/* planet left */}
            <Image
              src='/assets/images/planet_3.svg'
              alt='planet'
              position='absolute'
              left={{ base: '-25', md: 'none' }}
              bottom={{ base: '-10', md: 'none' }}
              overflow='hidden'
              zIndex={3}
              loading='lazy'
            />
            {/* planet right */}
            <Image
              src='/assets/images/planet_4.svg'
              alt='planet'
              w={{ base: '100%', md: '70%' }}
              position='absolute'
              right={{ base: '-25', md: 'none' }}
              top={{ base: '-10', md: 'none' }}
              overflow='hidden'
              zIndex={3}
              loading='lazy'
            />
            {/* Flare top */}
            <Image
              src='/assets/images/flare_5.svg'
              alt='flare'
              h='full'
              w='full'
              position='absolute'
              top={{ base: '-205', md: 'none' }}
              zIndex={3}
              loading='lazy'
            />
            {/* Flare bottom */}
            <Image
              src='/assets/images/flare_4.svg'
              alt='flare'
              w='full'
              position='absolute'
              bottom={{ base: '-90', md: 'none' }}
              zIndex={3}
              loading='lazy'
            />
          </>
        )}
```

is there any solution for this way?

Reply:
nvm maybe separating each function would be helpful

Reply:
yup its okay to create separate function. just make sure the rendering condition cover for all breakpoint

Reply:
one more from me, could you attach SS for this "iPad Air" screen? 820 x 1180 resolution

just want to make sure all breakpoints are covered

<img width="669" alt="image" src="https://github.com/user-attachments/assets/44d2c6ae-0a42-4083-93b1-23b354efe18f">

Reply:
![image](https://github.com/user-attachments/assets/d3be19a4-6baf-49f1-a81a-d106ae2b75b9)

![image](https://github.com/user-attachments/assets/a62e0e04-bc99-441f-9708-d79a0a603653)

@Fakhri1999

Reply:
nicee :rocket: